values.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html lang="en">
+	<!-- This file is a placeholder for a page. Rename this file to the name of the page you are working on. -->
+
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>If These Lands Could Talk - Our Values</title>
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@400;500;700&display=swap"
+			rel="stylesheet"
+		/>
+		<link rel="stylesheet" href="css/normalize.css" />
+		<link rel="stylesheet" href="css/global.css" />
+		<!-- In a new branch, rename this page.css file to the name of that actual page (Example: home.css) -->
+		<link rel="stylesheet" href="css/values.css" />
+	</head>
+
+	<body>
+		<!-- Header section -->
+		<!-- add aria labels -->
+		<section class="hero">
+			<div>
+				<!-- <source media="(min-width:768px)" srcset ="./assets/images/pages/values/hero-values.jpg" alt="a young man smiling"> -->
+				<!-- Not sure if desktop img needs new line  -->
+				<img
+					src="./assets/images/pages/values/mobile/hero-values-m.jpg"
+					alt="a young man smiling"
+				/>
+			</div>
+			<div class="values-main">
+				<div class="values-head">
+					<h3 class="h1-black">Our Values</h3>
+					<p>
+						We believe in equality and belonging for native and indigenous
```

Comment:
This text should be left-aligned on desktop. Limit the width as well. 

![Uploading Screen Shot 2024-01-19 at 11.18.25 AM.png…]()

Reply:
Done!